services/apps/cache_worker/src/workflows/spawnOrganizationAggregatesComputation.ts

```diff
@@ -0,0 +1,54 @@
+import {
+  ChildWorkflowCancellationType,
+  ParentClosePolicy,
+  continueAsNew,
+  executeChild,
+  proxyActivities,
+  workflowInfo,
+} from '@temporalio/workflow'
+
+import * as activities from '../activities/computeAggs/organization'
+
+import { processOrganizationAggregates } from './processOrganizationAggregates'
+
+const activity = proxyActivities<typeof activities>({ startToCloseTimeout: '15 minutes' })
+
+/*
+spawnOrganizationAggregatesComputation is a Temporal workflow that:
+  - [Activity]: Get organization IDs from Redis.
+  - [Child Workflow]: Re-compute and update aggregates for each organization 
+    in batches of 100. Child workflows run independently and won't be 
+    cancelled if the parent workflow stops.
+*/
+export async function spawnOrganizationAggregatesComputation(): Promise<void> {
+  const { organizationIds, totalSize } = await activity.getOrganizationIdsFromRedis()
+
+  if (!totalSize) {
```

Comment:
@sausage-todd The workflow processes organizations in random batches of 100. After each organization is processed, it is removed from the Redis set `organizationIdsForAggComputation`.

Since processed organizations are continuously removed, the set gradually shrinks. Eventually, when `SRANDMEMBER` finds no more organizations in the set, the workflow naturally stops.